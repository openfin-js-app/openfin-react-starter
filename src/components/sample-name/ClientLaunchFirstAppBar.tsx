import * as React from 'react';
import IconButton from '@material-ui/core/IconButton';
import {makeStyles} from "@material-ui/styles";

import ThumbUpIcon from '@material-ui/icons/ThumbUp'

import {clientLaunchFirstAppBarComp as style} from "../../assets/jss/sample-name";

import appLogo from '../../assets/svg/app.svg';

const useStyles = makeStyles(style);

const ClientLaunchFirstAppBar:React.FunctionComponent<{}>=(
    props
)=>{

    const classes = useStyles({});

    return (<div className={classes.outmostContainer}>
        <img src={appLogo} className={classes.appLogoImg}/>
        <IconButton aria-label="ThumbUp" className={classes.testBtn} >
            <ThumbUpIcon fontSize="small" />
        </IconButton>
    </div>)
}

export default ClientLaunchFirstAppBar;