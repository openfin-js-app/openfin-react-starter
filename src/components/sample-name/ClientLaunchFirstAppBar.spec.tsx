import * as React from 'react';
import { createShallow, createMount } from '@material-ui/core/test-utils';
import toJson from 'enzyme-to-json';

import ClientLaunchFirstAppBar from './ClientLaunchFirstAppBar'

describe('ClientLaunchFirstAppBar',()=>{

    let shallow;
    let mount;

    beforeEach(() => {
        shallow = createShallow();
        mount = createMount();
    });

    afterEach(()=>{
        mount.cleanUp();
    })

    it('render correctly by default',()=>{
        const wrapper = shallow(
            <ClientLaunchFirstAppBar/>
        )

        expect(toJson(wrapper)).toMatchSnapshot();
    })

})