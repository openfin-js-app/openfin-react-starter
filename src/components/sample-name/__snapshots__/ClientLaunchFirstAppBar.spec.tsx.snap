// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientLaunchFirstAppBar render correctly by default 1`] = `
<div
  className="makeStyles-outmostContainer-1"
>
  <img
    className="makeStyles-appLogoImg-2"
    src="app.svg"
  />
  <WithStyles(ForwardRef(IconButton))
    aria-label="ThumbUp"
    className="makeStyles-testBtn-3"
  >
    <ThumbUpIcon
      fontSize="small"
    />
  </WithStyles(ForwardRef(IconButton))>
</div>
`;
